### PR TITLE
Bug 1146400 - [NFC] Remove mozNfc checks from NfcManager, small clean up. r=gweng

### PR DESCRIPTION
**Pull request by tauzen**

### CODE REVIEW

File: apps/system/js/nfc_manager.js [hunk 1]
```diff
@@ -254,10 +268,7 @@
      * @memberof NfcManager.prototype
      */
     _checkP2PRegistration: function nm_checkP2PRegistration() {
-      var nfcdom = window.navigator.mozNfc;
-      if (!nfcdom) {
-        return;
-      }
+      var nfc = window.navigator.mozNfc;
```
Comment (snowmantw) [inline apps/system/js/nfc_manager.js]: So did we do something to make sure 'mozNfc' is always there?

File: apps/system/js/nfc_manager.js [hunk 2]
```diff
@@ -304,14 +325,12 @@
      * @memberof NfcManager.prototype
      */
     _dispatchP2PUserResponse: function nm_dispatchP2PUserResponse() {
-      var nfcdom = window.navigator.mozNfc;
-      if (!nfcdom) {
-        return;
-      }
+      var nfc = window.navigator.mozNfc;
```
Comment (snowmantw) [inline apps/system/js/nfc_manager.js]: The same question.

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#z0kVDb9qSbGDHgz5tGPzmw

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4f78f12181b6fbe0e2490d865c7b1a0cdbbfa6c5

**Pull request merged**